CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/tasks/LookupPluginsTask.java

```diff
@@ -39,106 +32,32 @@
-                                final List<DataAccessPlugin> pluginList = plugins.get(pluginToRemoveType);
-                                IntStream.range(0, pluginList.size())
-                                        .filter(index -> pluginList.get(index).getClass().getName()
-                                                .equals(pluginToRemoveName))
-                                        .findFirst()
-                                        .ifPresent(index -> {
-                                            // Remove the overriden plugin
-                                            pluginList.remove(index);
-                                            LOGGER.log(Level.INFO,
-                                                    String.format("Removed data access plugin %s (%s) as it is overriden.",
-                                                            pluginToRemoveName, pluginToRemoveType)
-                                            );
-                                        });
-                            })
-                );
-        
+        final Map<String, List<DataAccessPlugin>> plugins = DataAccessUtilities.getAllPlugins();
+        // Remove hidden data access categories
+        if ((!DAV_CATS.isEmpty()) && (!DAV_CATS.isBlank())) {
+            final String[] arrayOfcategory = addCategoryToList(DAV_CATS);
+            if (arrayOfcategory.length > 0) {
+                for (int i = 0; i < arrayOfcategory.length; i++) {
+                    plugins.remove(arrayOfcategory[i].trim());
+                }
+            }
+        }
         return plugins;
     }
-    
-    /**
-     * Logs that a data access plugin has been discovered and added to the plugin map.
-     * The log will indicate which type the plugin has been added under, in this case
-     * it is the actual plugins type.
-     *
-     * @param plugin the plugin that is being added
-     */
-    private void logDiscoveredDataAccessPlugin(final DataAccessPlugin plugin) {
-        logDiscoveredDataAccessPlugin(plugin, plugin.getType());
-    }
-    
-    /**
-     * Logs that a data access plugin has been discovered and added to the plugin map.
-     * The log will indicate which type the plugin has been added under, in this case
-     * it is passed type value.
-     *
-     * @param plugin the plugin that is being added
-     * @param pluginType the plugin type that the plugin will be added under
-     */
-    private void logDiscoveredDataAccessPlugin(final DataAccessPlugin plugin,
-                                               final String pluginType) {
-        LOGGER.log(Level.INFO, String.format("Discovered data access plugin %s (%s)",
-                                plugin.getName(), pluginType));
+
+    public static String[] addCategoryToList(final String categories) {
+        if ((!categories.trim().isEmpty()) && (!categories.trim().isBlank())) {
```

Comment:
Can simplify this if statement to `StringUtils.isNotBlank(categories)`.

Worth noting for future reference that `isBlank` is `isEmpty` or any other series of just whitespace characters (in other words, you only need to check for one of emptiness or blankness depending on your use case, and you don't need to trim before a blank check)